db/seeds.rb

```diff
@@ -557,6 +557,28 @@
 
 ##############################################################################
 
+num_of_display_ads = 2 * SEEDS_MULTIPLIER
+
+seeder.create_if_none(DisplayAd) do
+  orgs_in_random_order = Organization.order(Arel.sql("RANDOM()"))
+  orgs = orgs_in_random_order.to_a
+
+  num_of_display_ads.times do
+    DisplayAd.create!(
+      organization: orgs.sample,
+      name: Faker::Lorem.sentence,
+      body_markdown: "This is an external ad",
+      published: true,
+      approved: true,
+      placement_area: "post_comments"
```

Comment:
I'm thinking as a next step it would be useful to loop through some of the different placement areas when we creating the display ads.

Reply:
Actually I can do this now.